lib/resources/example_json/r4_examples_charge_item.rb

```diff
@@ -47,6 +47,15 @@ module Resources
               'valueCodeableConcept': {
                 'coding': [
                   {
+                    'extension': [
```

Comment:
Can we actually use chargeItem id 292870306 for our chargeItem example? It is a real example out of C1941.

Reply:
updated here with ChargeItem id 292870306
https://github.com/cerner/fhir.cerner.com/pull/629/commits/fe4639617def7400eddd5812dda0ac9a03a97c58

Reply:
+1 to new example